config/jobconfig.toml

```diff
@@ -1,10 +1,11 @@
 ## This is the configuration file for the job system.
 ## This will only be enabled when the config flag LOAD_JOBS_FROM_TXT is enabled.
 ## We use a system of keys here that directly correlate to the job, just to ensure they don't desync if we choose to change the name of a job.
-## You are able to change (as of now) four different variables in this file.
+## You are able to change (as of now) five different variables in this file.
 ## Total Positions are how many job slots you get in a shift, Spawn Positions are how many you get that load in at spawn. If you set this to -1, it is unrestricted.
 ## Playtime Requirements is in minutes, and the job will unlock when a player reaches that amount of time.
 ## However, that can be superseded by Required Account Age, which is a time in days that you need to have had an account on the server for.
+## Required Character Age checks if your character is sutable for this job. 0 = disabled.
```

Comment:
This shouldn't be updated here, this should be updated in the variable `config_documentation` on SSjob. Otherwise, this will clear out every time someone autogenerates it.

Reply:
Yeah, I see that.